tests/lib/docker.py

```diff
@@ -47,6 +47,7 @@ def run_fg(self, image, **kwargs):
         try:
             output = self._run(image=image, detach=False, stderr=True, **kwargs)
             print(str(output, "utf-8"))
+            return output
```

Comment:
I think that printing the output as well as returning it is likely to result in noisy tests.